rocketmq-docker/4.2.0/sample/play.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+docker build -t apache/rocketmq-namesrv:4.2.0-dev ./namesrv
+
+docker build -t apache/rocketmq-broker:4.2.0-dev ./broker
+
+docker run -d -p 9876:9876 --name rmqnamesrv  apache/rocketmq-namesrv:4.2.0-dev
```

Comment:
@vongosling I think it depends on the user scenarios. For clusters, I think it's better to have individual images for namesrv and broker.

Reply:
yes,IMO, there is a rush need for cluster case.

Reply:
@vongosling However, for clusters scenario, this can often be a big topic. Generally speaking, for stateful applications like RocketMQ running on Kubernetes we will need to write codes to implement the operation logics for them, including cluster nodes scale, auto or on-demand backup, storage, namesrv or broker recovery, etc. To achieve this I suggest it would be better to start a new project like` apache/rocketmq-operator`. Currently I'm not sure if you team internal have this plan. Personally speaking I think it can be a very good use case for community... Anyway, if you need someone to write a archetype for this operator I would be happy to do.

Huan Wei

Reply:
@XiaoZYang Agree, I've been thinking these days, It's time to start RIP(RocketMQ Improvment Proposal) in Community. @huanwei thoughts ?

Reply:
We should consider the compatibility of JVM options of default docker environment in next release, or we need to add a profile for Docker. 
What's your opinion? @huanwei @vongosling 
Because I think the fix scripts like `runbroker-dev.sh` is not very graceful.

Reply:
@XiaoZYang Agree. We need to dynamically get the available JVM heap size of the node, then set the JAVA_OPT accordingly by best-practise suggestion before rocket starts.

Reply:
@vongosling @XiaoZYang I just refactored this PR. Pls kindly review. Thanks.

Reply:
@vongosling @XiaoZYang 
Could you guide me how to participate in the community developer mailing list? Actually I have submitted the CLA to secretary and have received the response email on July 27, but I have no idea how to go ahead to contact the (P)PMC with my preferred Apache id `huanwei`. Thanks.

Reply:
@huanwei see http://rocketmq.apache.org/about/contact/ for mail list detials